src/Packlist/usePacklistForm.js

```diff
@@ -1,15 +1,20 @@
 import { useState } from 'react'
+import usePacklists from './usePacklists'
 
-export default function usePacklistForm(addPacklist) {
+export default function usePacklistForm() {
   const [packlistFormIsVisible, setPacklistFormIsVisible] = useState(false)
+  const { updatePacklist, addPacklist } = usePacklists()
 
   const showPacklistForm = () => setPacklistFormIsVisible(true)
 
   const onPacklistSave = (packlist) => {
     addPacklist(packlist)
     setPacklistFormIsVisible(false)
   }
-
+  const onPacklistSaveEdit = (packlist) => {
```

Comment:
Is this function necessary? It looks like all it does is call updatePacklist. If that's true, you might just call updatePacklist directly.

Reply:
I will proof it, because of using hook forms, at some points I need to handle functions more complicate than normally

Reply:
I need the return of true and it's easier to understand thats an edit function instead of just an update?